components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -123,7 +146,11 @@ class MozillaSocorroService(
         val gzipOs = GZIPOutputStream(os)
         sendPart(gzipOs, boundary, "ProductName", appName, nameSet)
         sendPart(gzipOs, boundary, "ProductID", appId, nameSet)
-        sendPart(gzipOs, boundary, "Version", version, nameSet)
+        sendPart(gzipOs, boundary, "Version", versionName, nameSet)
+        sendPart(gzipOs, boundary, "AcVersion", AcBuild.version, nameSet)
+        sendPart(gzipOs, boundary, "GleanVersion", AcBuild.gleanSdkVersion, nameSet)
+        sendPart(gzipOs, boundary, "AsVersion", AcBuild.applicationServicesVersion, nameSet)
```

Comment:
I would spell this out and not abbreviate it.

Reply:
Agreed.